api/nakadi-event-bus-api.yaml

```diff
@@ -827,135 +685,118 @@ definitions:
 
       For concrete examples of what will be enforced by Nakadi see the objects BusinessEvent and
       DataChangeEvent below.
-    additionalProperties: true
 
   EventMetadata:
     type: object
-    description: |
-      Metadata for this Event.
-
-      Contains commons fields for both Business and DataChange Events. Most are enriched by Nakadi
-      upon reception, but they in general MIGHT be set by the client.
     properties:
-      metadata:
```

Comment:
The `metadata` field is now declared as a property of the BusinessEvent and DataChangeEvent object. This allows the Metadata object's property fields to be direct children of the definition, making them visible to the ui and producing a better output from codegen. The definition of Metadata remain the same as do the actual JSON structures for BusinessEvent and DataChangeEvent.